graylog2-server/src/main/java/org/graylog2/rest/resources/system/lookup/LookupTableResource.java

```diff
@@ -173,9 +176,29 @@ public LookupTableResource(DBLookupTableService dbTableService,
         this.cacheSearchQueryParser = new SearchQueryParser(CacheDto.FIELD_TITLE, CACHE_SEARCH_FIELD_MAPPING);
     }
 
+    private void checkLookupTableId(String idOrName, LookupTableApi toUpdate) {
+        requireNonNull(idOrName, "idOrName parameter cannot be null");
+        if (!idOrName.equals(toUpdate.id()) || !idOrName.equals(toUpdate.name())) {
```

Comment:
Agreed. Changing the name will cause havoc anyway because we do not check if it's used...